### ISSUE TITLE
anError behavior

### ISSUE THREAD

**Issue by xhit** [mpetroff/pannellum#740]
Hello Matthew Petroff, first thanks for the great library! It's very useful!

I'm programming a tour's builder with **Pannellum**, and I need to avoid that user see **anError** method message because it break the continuity creating the tour.

The user put hotspots for every scene, but when a hotspot with type scene is called in the builder I get error <No panorama image was specified.> This is because I only permit add hotspots scene by scene and is good . I avoid this behavior adding this code block in method **createHotSpot** creating a new eventListener **customerror**

**OLD CODE**
`loadScene(hs.sceneId, hs.targetPitch, hs.targetYaw, hs.targetHfov);`

**NEW CODE**
```
var scenes = [];
for(var k in config.scenes) scenes.push(String(k));

if(scenes.indexOf(hs.sceneId) > -1){
         loadScene(hs.sceneId, hs.targetPitch, hs.targetYaw, hs.targetHfov);
}else{
        div.clicked = false;
        fireEvent('customerror',config.strings.noPanoramaError);
}
```

The code put in array every scene in config and check if sceneId selected is in the array, if not I fire a new event called **customerror** (temp event name) and set div.clicked false

**And in the web app I set the listener:**
```
v.on('customerror',function(event){
    alert(event);
 });
```

When an error happens I can do an action avoiding the user restart the web because the viewer is showing nothing.

I read the API and I can't clear the error without avoiding a reload for the viewer, so was a good idea create a new eventListener.

**Comment by mpetroff** [comment]
Is there a question here? I don't see one.

**Comment by xhit** [comment]
Sorry, let me explain. Instead show an error in Pannellum viewer that need a refresh in the website or refresh in the viewer, is better to handle the error and return it in an event like my customerror event, so a programmer can handle it and set the correct behavior in the viewer.

I failed, is not a question, this was my first interaction in github and I know about Pull Request. I created a fork that explain this behavior and I will consider add this option after the test in the builder.
